deploy/infrastructure/common/ecr.tf

```diff
@@ -10,3 +10,10 @@ module "ecr_ue2" {
   ]
   tags = local.tags
 }
+
+module "ecr_public_ue1" {
```

Comment:
Avoid `module` considering it contains a single resource and LOC would be halved without it. Instead, use the `resource` straight up.